packages/javascript/bh-shared-ui/src/components/NoDataAlert/NoDataAlert.tsx

```diff
@@ -20,25 +20,38 @@ import { FC } from 'react';
 type NoDataAlertProps = {
     dataCollectionLink: JSX.Element;
     fileIngestLink?: JSX.Element;
+    sampleDataLink?: JSX.Element;
 };
 
-export const NoDataAlert: FC<NoDataAlertProps> = ({ dataCollectionLink, fileIngestLink }) => {
+export const NoDataAlert: FC<NoDataAlertProps> = ({ dataCollectionLink, fileIngestLink, sampleDataLink }) => {
     const theme = useTheme();
 
     return (
         <Box display={'flex'} justifyContent={'center'} mt={theme.spacing(8)} mx={theme.spacing(4)}>
             <Alert severity={'info'}>
                 <AlertTitle>No Data Available</AlertTitle>
-                It appears that no data has been uploaded yet.
-                <br />
-                See our {dataCollectionLink} documentation to learn how to start collecting data.
-                <br />
-                {fileIngestLink && (
-                    <>
-                        If you have files available from a SharpHound or AzureHound collection, please visit the{' '}
-                        {fileIngestLink} page to begin uploading your data.
-                    </>
-                )}
+                <p>
+                    It appears that no data has been uploaded yet. See our {dataCollectionLink} documentation to learn
+                    how to start collecting data.
+                </p>
+
+                <p>
```

Comment:
We may want to move these paragraphs within the block, since the block is conditional and we probably won't want empty paragraphs when they're not included